site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -174,6 +174,9 @@ Response:
 {"result":{"object_meta":{"name":"local","namespace":"default","uid":"1234","resource_version":"v1","generation":"1","creation_timestamp":"1533766607","annotations":{"annotation":"true"},"labels":{"islocal":"true"}},"status":{"state":"Ready","address":"127.0.0.1","ports":[{"name":"default","port":7777}]}}}
 {"result":{"object_meta":{"name":"local","namespace":"default","uid":"1234","resource_version":"v1","generation":"1","creation_timestamp":"1533766607","annotations":{"annotation":"true"},"labels":{"islocal":"true"}},"status":{"state":"Ready","address":"127.0.0.1","ports":[{"name":"default","port":7777}]}}}
 ```
+
+The Watch GameServer stream is also exposed on the same `/watch/gameserver` URL as a WebSocket endpoint using the same newline-delimited JSON format for clients that support WebSocket but not HTTP streaming, such as Unreal.
```

Comment:
Yay docs!

Can we wrap this in a `feature` shortcode (details on how to do that [here](https://agones.dev/site/docs/contribute/#documentation-for-upcoming-features)) so it only shows up in 1.13.0

Do we want to also be explicit that it's the same port as the HTTP rest api, etc. just to make it super, super clear.

Apparently you can test websockets with `curl` ([gist](https://gist.github.com/htp/fbce19069187ec1cc486b594104f01d0)) - shall we also include an example, that would also be awesome.